src/editor/ui_menus/scenario_infrastructure_settings_window.cc

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (C) 2002-2019 by the Widelands Development Team
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ */
+
+#include "editor/ui_menus/scenario_infrastructure_settings_window.h"
+
+#include <memory>
+
+#include <boost/format.hpp>
+
+#include "base/i18n.h"
+#include "economy/ware_instance.h"
+#include "editor/editorinteractive.h"
+#include "editor/tools/scenario_infrastructure_settings_tool.h"
+#include "logic/map_objects/tribes/tribes.h"
+#include "logic/map_objects/tribes/ware_descr.h"
+#include "wui/economy_options_window.h"
+
+inline EditorInteractive& ScenarioFlagSettingsWindow::eia() {
+	return dynamic_cast<EditorInteractive&>(*get_parent());
+}
+
+ScenarioFlagSettingsWindow::ScenarioFlagSettingsWindow(EditorInteractive& parent,
+                                                       ScenarioInfrastructureSettingsTool& t,
+                                                       Widelands::Flag& f)
+   : UI::Window(
+        &parent,
+        "scenario_flag_settings_" + std::to_string(f.serial()),
+        0,
+        0,
+        300,
+        100,
+        (boost::format(_("Flag at %1$dx%2$d")) % f.get_position().x % f.get_position().y).str()),
```

Comment:
```suggestion
        (boost::format(_("Flag at %1%×%2%")) % f.get_position().x % f.get_position().y).str()),
```
* I think `%1%` is easier to read for translators than `%1$d`
* Unicode multiplication sign character